snap/snapcraft.yaml

```diff
@@ -16,16 +16,27 @@ confinement: strict
 compression: lzo
 
 parts:
-  amber:
+  amber-bash:
     plugin: rust
     source: .
+    build-packages:
+      - yq
```

Comment:
what is this? isnt this PR about getting version from the cargo file?

Reply:
Yeah, that's what it does. `jq` has the binary `tomlq` in it which makes it easy to read the version